File: orto-protesi/src/app/models/LavorazioneDTO.ts
export interface LavorazioneDTO {
  id: number;
  clienteId?: number;
  creatoDaUtenteId?: number;
  nomePaziente: string;
  cognomePaziente: string;
  etaPaziente?: number | null;
  numeroTelefonoPaziente?: string;
  codiceFiscalePaziente?: string;
  nomeCompletoCliente?: string;
  tipiLavorazione: string[];
  stato: string;
  specificheTecniche?: string;
  dataCreazione?: Date | string;
  percorsiFileImmagini?: string[];
  percorsiFileStl?: string[];
}

File: OrtoProtesiApi/DTOs/LavorazioneDTO.cs

namespace OrtoProtesiApi.DTOs
{
    public class LavorazioneDTO
    {
        public int Id { get; set; }
        public int? ClienteId { get; set; }
         public int CreatoDaUtenteId { get; set; }
        public string NomePaziente { get; set; } = string.Empty;
        public string CognomePaziente { get; set; } = string.Empty;
        public int? EtaPaziente { get; set; }
        public string? NumeroTelefonoPaziente { get; set; } = string.Empty;
        public string? CodiceFiscalePaziente { get; set; } = string.Empty;
        public string NomeCompletoCliente { get; set; } = string.Empty;
        public List<string> TipiLavorazione { get; set; } = new();
        public string Stato { get; set; } = string.Empty;
        public string SpecificheTecniche { get; set; } = string.Empty;
        public DateTime DataCreazione { get; set; } = DateTime.Now;

        public List<string> PercorsiFileImmagini { get; set; } = new();
        public List<string> PercorsiFileStl { get; set; } = new();
    }
}